scripts/build_docs.sh

```diff
@@ -39,7 +39,11 @@ sed -i s/TOOLCHAIN/`grep -oP 'v4\..*' lean-toolchain`/ docbuild/lakefile.toml
 cd docbuild
 
 # Disable an error message due to a non-blocking bug. See Zulip
```

Comment:
imo this message isn't at all helpful, and we should replace it with a link to something like this: https://github.com/leanprover/doc-gen4?tab=readme-ov-file#usage since point 3 is the one saying to do this

Reply:
I agree. I didn't add that btw :)

Reply:
I had no intention of blaming you :)